traefik/Changelog.md

```diff
@@ -1,5 +1,219 @@
+     # -- Enables accessLogs for internal resources. Default: false.
+     addInternals: false
+     fields:
+@@ -566,24 +581,16 @@ additionalArguments: []
+ #  - "--providers.kubernetesingress.ingressclass=traefik-internal"
+ #  - "--log.level=DEBUG"
+ 
+-# -- Environment variables to be passed to Traefik's binary
++# -- Additional Environment variables to be passed to Traefik's binary
+ # @default -- See _values.yaml_
+-env:
+-- name: POD_NAME
+-  valueFrom:
+-    fieldRef:
+-      fieldPath: metadata.name
+-- name: POD_NAMESPACE
+-  valueFrom:
+-    fieldRef:
+-      fieldPath: metadata.namespace
++env: []
+ 
+ # -- Environment variables to be passed to Traefik's binary from configMaps or secrets
+ envFrom: []
+ 
+ ports:
+   traefik:
+-    port: 9000
++    port: 8080
+     # -- Use hostPort if set.
+     hostPort:  # @schema type:[integer, null]; minimum:0
+     # -- Use hostIP if set. If not set, Kubernetes will default to 0.0.0.0, which
+@@ -601,7 +608,7 @@ ports:
+     expose:
+       default: false
+     # -- The exposed port for this service
+-    exposedPort: 9000
++    exposedPort: 8080
+     # -- The port protocol (TCP/UDP)
+     protocol: TCP
+   web:
+@@ -614,7 +621,7 @@ ports:
+       default: true
+     exposedPort: 80
+     ## -- Different target traefik port on the cluster, useful for IP type LB
+-    targetPort:  # @schema type:[integer, null]; minimum:0
++    targetPort:  # @schema type:[string, integer, null]; minimum:0
+     # The port protocol (TCP/UDP)
+     protocol: TCP
+     # -- See [upstream documentation](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport)
+@@ -653,7 +660,7 @@ ports:
+       default: true
+     exposedPort: 443
+     ## -- Different target traefik port on the cluster, useful for IP type LB
+-    targetPort:  # @schema type:[integer, null]; minimum:0
++    targetPort:  # @schema type:[string, integer, null]; minimum:0
+     ## -- The port protocol (TCP/UDP)
+     protocol: TCP
+     # -- See [upstream documentation](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport)
+@@ -780,8 +787,8 @@ autoscaling:
+ 
+ persistence:
+   # -- Enable persistence using Persistent Volume Claims
+-  # ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+-  # It can be used to store TLS certificates, see `storage` in certResolvers
++  # ref: http://kubernetes.io/docs/user-guide/persistent-volumes/.
++  # It can be used to store TLS certificates along with `certificatesResolvers.<name>.acme.storage`  option
+   enabled: false
+   name: data
+   existingClaim: ""
+@@ -797,7 +804,7 @@ persistence:
+ # -- Certificates resolvers configuration.
+ # Ref: https://doc.traefik.io/traefik/https/acme/#certificate-resolvers
+ # See EXAMPLES.md for more details.
+-certResolvers: {}
++certificatesResolvers: {}
+ 
+ # -- If hostNetwork is true, runs traefik in the host network namespace
+ # To prevent unschedulabel pods due to port collisions, if hostNetwork=true
```

Comment:
```suggestion
 # To prevent unschedulable pods due to port collisions, if hostNetwork=true
```

Reply:
Good catch ! This one has to be done on values.yaml, not this PR. It will be for the next release.

Reply:
alright!

Reply:
I opened this PR  to fix it: https://github.com/traefik/traefik-helm-chart/pull/1241